recipe/meta.yaml

```diff
@@ -27,31 +28,30 @@ requirements:
   host:
     - pip
     - python
-    - numpy {{ numpy }}
```

Comment:
Out of curiosity, why did we need to change this from the `cbc.yaml` `numpy` version to a pinned version?

Reply:
spacy and a few of its dependencies require numpy >=2,<3 to build the package. Our base cbc.yaml pins numpy to 1.2x versions but they are incompatible